modules/dag-l1/src/main/scala/org/tessellation/dag/l1/domain/trust/storage/TrustStore.scala

```diff
@@ -0,0 +1,12 @@
+package org.tessellation.dag.l1.domain.trust.storage
+
+import org.tessellation.schema.peer.PeerId
+import org.tessellation.schema.trust.TrustDbValues
+
+trait TrustStore[F[_]] {
```

Comment:
`Store` -> `Storage` to keep the same naming within the codebase

Reply:
or `TrustDbStorage` as indeed it's a database storage, not SQL storage :D

Reply:
Re-named `TrustSqlStorage` back to `TrustDbStorage`.